text/0000-launcher-sbom.md

```diff
@@ -0,0 +1,160 @@
+
+* If no SBOM files are found in `/cnb/lifecycle` (e.g., if the builder author did not include them), the lifecycle will warn and
+  continue. Alternatively, the lifecycle could generate the files on the fly, but this would increase build times.
+    * If there is a previous image, `<layers>/sbom/launch/buildpacksio_lifecycle` may contain SBOM files - but these
+      should be deleted before `export` as there is no guarantee that the lifecycle that created the previous image is
+      the same as the current lifecycle.
+
+* In theory, there should be no changes needed for end-users to consume the new SBOM files, as the files will be placed
+  in the same directory, with the same structure, as SBOM files for buildpack-provided dependencies.
+
+# Drawbacks
+
+[drawbacks]: #drawbacks
+
+Why should we *not* do this? More work for the lifecycle.
+
+# Alternatives
+
+[alternatives]: #alternatives
+
+- What other designs have been considered?
+    - This could all be implemented with a buildpack - e.g., a `buildpacksio/lifecycle` utility buildpack whose sole
+      responsibility would be to copy the SBOM files from `/cnb/lifecycle` to `<layers>/sbom`. However, this would
+      introduce quite a bit of complexity and overhead for what is ultimately a very simple operation. A potential
+      benefit is that platforms using Platform API `0.8` and higher wouldn't need to upgrade - but they would need to
+      add the new utility buildpack to builders, which is probably just as much work.
+    - In theory, we don't actually need the files to be in `<layers>/sbom/<build|launch>/buildpacksio_lifecycle` -
+      because any files restored from a previous build will be ignored (see above). They could be annotations or
+      attestations on the image instead (see https://github.com/buildpacks/rfcs/pull/195) - but this would require
+      platforms to keep track of the files as inputs to the signer binary.
+
+- Why is this proposal the best? It is easy and straightforward for the lifecycle to copy pre-generated files
+  to `<layers>/sbom/<build|launch>/buildpacksio_lifecycle`.
+
+- What is the impact of not doing this? A less complete SBOM for CNB-built images.
+
+# Prior Art
+
+[prior-art]: #prior-art
+
+Discuss prior art, both the good and bad.
+
+* RFCs:
+    * Base image dependencies: https://github.com/buildpacks/rfcs/pull/211, https://github.com/buildpacks/rfcs/pull/195
+    * Buildpack-provided dependencies: [RFC 0095](https://github.com/buildpacks/rfcs/blob/main/text/0095-sbom.md)
+
+# Unresolved Questions
+
+[unresolved-questions]: #unresolved-questions
+
+- What parts of the design do you expect to be resolved through implementation of the feature?
+    - Tooling or libraries used by the lifecycle to generate SBOM files describing itself.
+
+# Spec. Changes (OPTIONAL)
+
+[spec-changes]: #spec-changes
+Does this RFC entail any proposed changes to the core specifications or extensions? If so, please document changes here.
```

Comment:
Actually, not sure - the SBOM files would be in locations that are already spec'd, and it seems that `buildpacksio` is already a reserved namespace.

Reply:
Discussed in 8/17 Working Group - these changes would require a new platform API, but not a new buildpack API.